packages/js-marker-clusterer/services/managers/cluster-manager.ts

```diff
@@ -139,6 +140,14 @@ export class ClusterManager extends MarkerManager {
     });
   }
 
+  createClusterEventObservable<T>(eventName: string): Observable<T> {
+    return Observable.create((observer: Observer<T>) => {
+      this._clustererInstance.then((m: MarkerClustererInstance) => {
```

Comment:
this._clustererInstance.then should be run outside of angular zone.

Reply:
I believe this was copied from `packages/core/services/managers/info-window-manager.ts` or one of the other files. Should those also run outside of the angular zone? I don't really know much about this PR, but it seemed to follow the other managers and seemed like it could be merged in so we don't have as many stale PRs in this repo